core/internal/features/features_test.go

```diff
@@ -352,7 +434,6 @@ func TestIntegration_DirectRequest(t *testing.T) {
 	for _, tt := range tests {
 		test := tt
 		t.Run(test.name, func(t *testing.T) {
-			t.Parallel()
```

Comment:
Note: I had to remove this to get the tests to pass reliably. Because each test is setting up its own simulator, making them run in parallel is causing both simulators to race when adding transactions, and seems to be causing a deadlock somewhere. I want to keep momentum up so am not going to tackle that particular issue at this time.